src/api/Jalasoft.TeamUp.Resumes.API/Controllers/GetPostulations.cs

```diff
@@ -48,7 +49,7 @@ public GetPostulations(IPostulationsService postulationsService)
                     result = this.postulationsService.GetPostulationsByProjectId(projectId);
                 }
 
-                if (result.Length == 0)
+                if (result.FirstOrDefault()?.Id == 0)
```

Comment:
question, what is the purpose of this? for example if the result is null is it ok that returns ok an goes outside the if?

Reply:
yes, that is right, Id = 0 means that the resume Id does not exist, otherwise it exists but has no postulations